### PR TITLE
Update libvulkan to 1.1.70+dfsg1-1

### PR DESCRIPTION
**Pull request by craftyguy**

### CODE REVIEW

File: slave/packages.sls [hunk 1]
```diff
@@ -102,9 +102,9 @@ runner-packages:
             - libudev-dev:i386
             - libvdpau-dev
             - libvdpau-dev:i386
-            - libvulkan-dev
-            - libvulkan1
-            - libvulkan1:i386
+            - libvulkan-dev: 1.1.70+dfsg1-1
+            - libvulkan1: 1.1.70+dfsg1-1
+            - libvulkan1:i386: 1.1.70+dfsg1-1
```
Comment (dcbaker) [inline slave/packages.sls:107]: I'm not seeing where the second `:` comes from, is that an apt or salt syntax?

Comment (craftyguy) [reply slave/packages.sls:107]: a salt thing, I think. I followed this: https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.installed

**Comment by craftyguy** [comment]
Closing since libvulkan is in debian testing now, and we don't want to restrict the version in config if we don't have to.

File: slave/packages.sls (hunk 1 above)
Comment (dcbaker) [reply slave/packages.sls:107]: that was the documentation I couldn't find :)

looks good to me.

**Pull request closed without merging**